src/OrbitServiceProvider.php

```diff
@@ -59,8 +66,13 @@ public function boot()
             ], 'orbit:config');
         }
 
-        if (! Schema::connection('orbit')->hasTable('_orbit_meta')) {
-            Schema::connection('orbit')->create('_orbit_meta', function (Blueprint $table) {
+        if (! (new Filesystem())->exists($metaPath = Orbit::getMetaDatabasePath())) {
+            (new Filesystem())->ensureDirectoryExists(dirname($metaPath), 0755, true);
+            (new Filesystem())->put($metaPath, '');
```

Comment:
How about storing the `Filesystem` object in a variable so we don't have to new it up three times?

Reply:
Just used the `File::` facade instead.